src/SfsClient/readme.md

```diff
@@ -1,9 +1,9 @@
 ## SfsClient
 
-Do not change code under the sfs-client directory; it contains sfs-client source code from commit [be733af](https://github.com/microsoft/sfs-client/commits/be733af).  
+Do not change code under the sfs-client directory; it contains sfs-client source code from commit [c639a50](https://github.com/microsoft/sfs-client/commit/c639a50).  

```

Comment:
Please wait for the release of the client to happen before picking the commit hash.

Reply:
Do use this release https://github.com/microsoft/sfs-client/releases/tag/1.1.0